providers/tokenbucket/examples_test.go

```diff
@@ -0,0 +1,32 @@
+package tokenbucket
+
+import (
+	middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2"
+	grpc_ratelimit "github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/ratelimit"
+	"github.com/juju/ratelimit"
+	"google.golang.org/grpc"
+)
+
+// Hard-coded for simplicity sake, but make this configurable in your application.
+const (
+	// Add 5 token per seconds.
+	rate = 5
+	// Capacity of bucket. allow only 40 requests.
+	tokenCapacity = 40
+)
+
+// Simple example of server initialization code.
+func Example() {
+
+	limiter := TockenBucketInterceptor{}
```

Comment:
build error?

Reply:
it was a missed renaming from Yash , fixed thx.